docs/first-order/classify_waveforms.ipynb

```diff
@@ -0,0 +1,2327 @@
+{
```

Comment:
<p>In the first sentence, don't say "generate a list of data" from the units table. Be precise about what the output is. In this case, since it can be used for many kinds of data, you could be more explicit about that. This function actually outputs a list of <em>indices. </em>You mention this further down so its good, but maybe rephrase to first part to be clear that this could be used to access unit attributes rather than just "data".</p><p> </p><p>Typo in "for-loop that loops through the..."</p><p> </p><p>Typo in "superceeding"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>